cijoe/scripts/latency_plotter.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env python
+"""
+Extract and transform fio latency measurements
+=============================================
+
+fio has:
+
+* bandwidth in KiB / second
+* latency is mean and in nano-seconds
+* IOPS are 'raw' / base-unit
+
+.. note::
+    The metric-context need addition of backend-options, the options are
+    semi-encoded by ioengine and 'label', however, that is not very precise.
+
+.. note::
+    This uses matplotlib and numpy for plotting
+"""
+import errno
+import hashlib
+import json
+import logging as log
+import os
+import pprint
+import re
+import traceback
+from pathlib import Path
+from typing import Dict, List
+
+import matplotlib.pyplot as plt
+import numpy as np
+from cijoe.core.analyser import to_base_unit
+from cijoe.core.resources import dict_from_yamlfile
+from cijoe.fio.wrapper import dict_from_fio_output_file
+
+JSON_DUMP = {"indent": 4}
+
+FIO_COMPOUND_FILENAME = "fio-output-compound.json"
+FIO_OUTPUT_NORMALIZED_FILENAME = "fio-output-normalize.json"
+
+BDEVPERF_OUTPUT_PREFIX = "bdevperf-output_"
+BDEVPERF_OUTPUT_NORMALIZED_FILENAME = "bdevperf-output-normalized.json"
```

Comment:
If I'm not mistaken, we don't use the BDEVPERF at all, so any code related to it could be removed